packages/blade/src/_helpers/types.ts

```diff
@@ -83,6 +86,46 @@ type TestID = {
   testID?: string;
 };
 
+/**
+ * Returns the object as it is in web and sets all properties of object to type `never` in react native
+ */
+type MakeObjectWebOnly<T> = Platform.Select<{
```

Comment:
You're the new TS god 🙌

Reply:
Learning from the OG TS god Anurag  🙌🏼

Reply:
:godmode:

Reply:
can't find its usage? where is this used?

Reply:
Yup I removed it's usage later because now we can just use `PickCSSByPlatform<'display' | 'non-existing-prop'>` and that in itself removes props that are not supported in react native. 
 
Removing the utility as well. Can add it when we need.